src/PrizePool.sol

```diff
@@ -86,6 +88,9 @@ contract PrizePool {
     /// @notice Records the claim record for a winner
     mapping(address => ClaimRecord) internal claimRecords;
 
+    /// @notice Tracks the total fees accrued to each claimer
```

Comment:
nit: "Rewards" for Prize Claimers rather than "Fees" seems more appropriate. "Fees" makes me think the Claimer is losing that amount. This comment applies to the Claimer contract as well.